marshal/consumer.go

```diff
@@ -111,6 +111,10 @@ type Consumer struct {
 	rand       *rand.Rand
 	partitions map[string]int
 	claims     map[string]map[int]*claim
+
+	// Timestamp of when this consumer has been paused.
+	pauseExpireTime time.Time
+	adminID         string
```

Comment:
nit: Can this be like `pauseAdminID` or something so it's clear this is about the pause?